### DISCUSSION TITLE
Add a list of tags in the documentation

### DISCUSSION THREAD

**Discussion by plusiv** [brianvoe/gofakeit#348] [Ideas]
Would be great adding a list of all available tags for Struct fields and its mapped functions in the `README.md` file.

**Comment by brianvoe** [comment]
All of them as long as the output matches.

**Reply by plusiv** [reply]
In that case, there must be a mistake with `futuretime` and `pasttime` tags with `FutureData` and `PastTime` functions? 

https://github.com/brianvoe/gofakeit/blob/a5e6a8913e2926f59dd945e79f1298d9d18eb6ae/time.go#L324-L344

**Comment by brianvoe** [comment]
Output typo. fixing it now
